CONTRIBUTING.md

```diff
@@ -1,188 +1,31 @@
 # Contributing to MicroShift
 
-If you would like to develop MicroShift locally, you can follow this guide on getting
-it installed and running through the provided Makefile.
+Welcome to MicroShift! We are glad you are interested in this project. 
+Pull Requests are always welcome. If you’d like to jump right in with MicroShift, 
+check out the [open issues](https://github.com/redhat-et/microshift/issues). 
+No pull request is too small, we’re pretty sure you might see some typos in the code, 
+so please help us to fix anything you see!
 
-This guide will primarily focus on running MicroShift within a VM using Vagrant.
 
+## Documentation
 
-## (optional) Developing with Vagrant
+Looking for the docs? Well, look no further; [microshift.io](https://microshift.io) has the answers. 
+If the docs aren’t fulfilling enough, you can add to them by contributing in the [documentation repo](https://github.com/redhat-et/microshift-documentation).
 
-To get started with development, it is recommended to use Vagrant for VM provisioning,
-however it is not necessary.
 
-You can find a guide on how to install it for your system [here](https://www.vagrantup.com/downloads). 
+## Issues
 
-Once Vagrant is installed, you will need to create a Vagrant box for the operating
-system of your choice. For this example we will be looking at a [fedora 34 cloud
-image](https://app.vagrantup.com/fedora/boxes/34-cloud-base), however you can substitute any vagrant image of your choice.
+Notice a bug? File an issue [here](https://github.com/redhat-et/microshift/issues/new/choose) and provide as much information as possible.
 
+## Enhancement Proposals
 
-First, navigate to the MicroShift directory on your host system, or another designated
-directory where we will be storing the Vagrantfile.
+MicroShift’s goal is to create a minimal core that can functionally operate as close to OpenShift as 
+possible while maintaining a low resource impact. We are always looking for enhancement proposals, 
+so feel free to create a pull-request and an additional document explaining your proposal. 
```

Comment:
Instead of opening a PR, they should `create an enhancement issue with a link to a document that explains your proposal` 
Also, you can add
If the enhancement proposal aligns with [MicroShift's design goals]((https://github.com/redhat-et/microshift/blob/main/docs/design/design.md) it will be considered.